.github/workflows/build_docker_image.yml

```diff
@@ -0,0 +1,41 @@
+name: MultiArchDockerBuild
+
+on:
+  create:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build_multi_arch_image:
+    name: Build multi-arch Docker image.
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+        
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/epicgames-freegames-node:${{ github.sha }}
```

Comment:
This should be `${{ secrets.DOCKERHUB_USERNAME }}/epicgames-freegames:${{ github.sha }}`

Can you also add a latest tag, so:
`${{ secrets.DOCKERHUB_USERNAME }}/epicgames-freegames:${{ github.sha }},${{ secrets.DOCKERHUB_USERNAME }}/epicgames-freegames:latest`